redux-todomvc/store/configureStore.js

```diff
@@ -13,5 +16,7 @@ export default function configureStore(initialState) {
     })
   }
 
+  marionetteDispatch(store.dispatch, window.Backbone, window._);
```

Comment:
You have to pass in Backbone, Marionette, and underscore because your app old app may use globals, requirejs, webpack, etc. and we're not going to judge.